### ISSUE TITLE
New weird XFS debug output in our tests

### ISSUE THREAD

**Issue by mulkieran** [stratis-storage/devicemapper-rs#256]
I don't recall seeing it before today.

**Comment by mulkieran** [comment]
```
test thindev::tests::loop_test_snapshot_usage ... Clearing log and setting UUID
writing all SBs
new UUID = 328906fd-2482-49d5-a0b8-9a858ba4e518
```

**Comment by mulkieran** [comment]
Did they mean to do that?

**Comment by mulkieran** [comment]
Ok. It's just ```xfs_admin``` command-line call spitting out stuff in a snapshot usage test.

**Comment by mulkieran** [comment]
I think this is closed by #335.
